src/main/java/com/github/vkorobkov/jfixturescmd/Main.java

```diff
@@ -0,0 +1,16 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import com.github.lalyos.jfiglet.FigletFont;
+
+import java.io.IOException;
+
+public final class Main {
```

Comment:
Skipping JaCoCo execution due to missing execution data jfixtures-cmd/target/jacoco.exec